examples/swap/swap-e2e.html

```diff
@@ -105,12 +105,12 @@ <h3>Refund swap:</h3>
     /* global $, ChainAbstractionLayer, web3 */
     const { Client, providers, networks, crypto } = ChainAbstractionLayer
     const bitcoin = new Client()
-    bitcoin.addProvider(new providers.bitcoin.BitcoreRPCProvider('https://localhost:18332', 'bitcoin', 'local321'))
+    bitcoin.addProvider(new providers.bitcoin.BitcoreRPCProvider('http://localhost:18332', 'bitcoin', 'local321'))
```

Comment:
switch back to `https`

Reply:
I actually recall discussing whether https was required and i was under the impression that it was for some reason. I then tried to change to http and it still worked. So due to the default host port of starting a local rpc is http, i thought i could change it to that.